introduction/fr/06-conclusion.asciidoc

```diff
@@ -0,0 +1,13 @@
+== Conclusion
+
+Dans ce parcours d'apprentissage, nous avons donné une introduction à InnerSource.
+InnerSource applique les meilleures pratiques et les principes de l'open source au développement de logiciels internes.
+Il offre une option supplémentaire aux consommateurs lorsque les équipes de production ne sont pas en mesure de fournir une demande de fonctionnalité nécessaire.
+Un projet InnerSource réussi implique un https://innersourcecommons.org/learn/learning-path/product-owner/01[_product owner_] et un https://innersourcecommons.org/learn/learning-path/trusted-committer/01[_trusted committer_] de l'équipe *host* ainsi qu'un https://innersourcecommons.org/learn/learning-path/contributor/01[_contributor_] de l'équipe *guest*.
+Réalisée efficacement, InnerSource apporte de nombreux avantages aux deux équipes participantes.
+Les principes clés sur lesquels repose l'efficacité d'InnerSource sont la *contribution volontaire au code* et le *mentorat hiérarchisé*.
+
+Bien que cette formation donne un aperçu de haut niveau d'InnerSource, il existe de nombreux autres détails utiles pour faire fonctionner InnerSource pour votre équipe.
+Si vous souhaitez rester connecté à la conversation en cours autour d'InnerSource et de ses meilleures pratiques, alors rejoignez http://innersourcecommons.org [the InnerSource Commons].
```

Comment:
Si vous souhaitez rester connecté à la conversation en cours autour de l'InnerSource et de ses meilleures pratiques, alors rejoignez http://innersourcecommons.org [the InnerSource Commons].